<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iface</class>
 <widget class="QMainWindow" name="iface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>48</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>88</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>128</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>168</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>208</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>248</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>288</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>328</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Output buffer</string>
    </property>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>180</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create ACL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>150</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Upgrade Firmware</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>120</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Assign IP v4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>90</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create SNMP group</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Close Site</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Site</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>240</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reboot Remote Hardware</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>380</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aux BTN #2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>410</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aux BTN #3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>440</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aux BTN #4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>470</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aux BTN #5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>210</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>350</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aux BTN #1</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>471</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClear">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>510</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>C&amp;lear OUT buffer</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>530</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>E&amp;xit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>-1</y>
      <width>281</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Input string</string>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>lineEdit_5</zorder>
   <zorder>lineEdit_6</zorder>
   <zorder>lineEdit_7</zorder>
   <zorder>lineEdit_8</zorder>
   <zorder>pushButtonExit</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
